paragon/core/services/fe15_sprites.py

```diff
@@ -0,0 +1,169 @@
+        pid = self.gd.key(rid)
+
+        aid = self.gd.string(rid, "aid")
+        if aid:
+            return aid[4:]
+        else:
+            job, _ = self._person_to_jobs(rid)
+            return job[4:] if job else None
+
+    def _person_to_jobs(self, rid) -> Tuple[Optional[str], Optional[str]]:
+        job = self.gd.rid(rid, "class_1")
+        if not job:
+            return None, None
+        else:
+            jid = self.gd.key(job)
+            if not jid:
+                return None, None
+            else:
+                return jid, jid
+
+    def _load(self, char, job, team, fallback_job=None, animation=0) -> Optional[QPixmap]:
+        # Try to load the unique sprite
+        try:
+            sprite_filename = team + "1.bch.lz" if animation else team + "0.bch.lz" 
+
+            # Check for a unique sprite.
+            path = os.path.join("unit", "Unique", f"{job}_{char}")
+            path_2 = os.path.join("unit", "Unique", f"{job}_{job}")
+            anime_path = os.path.join(path, "anime.bin")
+            anime_path_2 = os.path.join(path_2, "anime.bin")
+            if self.gd.file_exists(anime_path, False):
+                # Found a unique sprite. Load it!
+                image_path = os.path.join(path, sprite_filename)
+                rid = self.gd.multi_open("sprite_data", anime_path)
+                return FE15SpriteModel(
+                    self._load_unique_sprite(image_path),
+                    self._load_animation_data(rid, animation=animation),
+                    team
+                )
+            elif self.gd.file_exists(anime_path_2, False):
+                # Found a unique sprite. Load it!
+                image_path = os.path.join(path_2, sprite_filename)
+                rid = self.gd.multi_open("sprite_data", anime_path_2)
+                return FE15SpriteModel(
+                    self._load_unique_sprite(image_path),
+                    self._load_animation_data(rid, animation=animation),
+                    team
+                )
+
+            # Extract data for building the sprite from its components.
+            body_path = os.path.join("unit", "Body", job)
+            anime_path = os.path.join(body_path, "anime.bin")
+            rid = self.gd.multi_open("sprite_data", anime_path)
+            sprite_data = self._load_animation_data(rid, animation=animation)
+
+            # Load by stitching together body and head sprites.
+            body_filename = os.path.join(body_path, sprite_filename)
+            head_path = os.path.join("unit", "Head", char)
+            head_filename = os.path.join(head_path, sprite_filename)
+
+            return FE15SpriteModel(
+                self._load_standard_sprite(sprite_data, body_filename, head_filename),
+                sprite_data,
+                team
+            )
+        except:
+            logging.exception("Failed to load sprite.")
+            raise
+
+    def is_vallite(self, person):
```

Comment:
Is this actually used in FE15? If not, there's a default implementation in the base class.

Reply:
My bad. I was c/p'ing as FE15's sprite loading logic is the same as FE14's. Will remove tomorrow.